src/components/calcite-tile-select/calcite-tile-select.e2e.ts

```diff
@@ -25,10 +25,10 @@ describe("calcite-tile-select", () => {
       { propertyName: "heading", value: "My Tile Select Heading" },
       { propertyName: "hidden", value: true },
       { propertyName: "icon", value: "layers" },
+      { propertyName: "inputAlignment", value: "end" },
```

Comment:
Nitpick: one value should suffice for this test. Just needs a valid value to do the check.

Reply:
Will remove this and other duplicate values in these tests.